pkg/yttlibrary/url.go

```diff
@@ -27,13 +27,23 @@ var (
 
 				"query_params_encode": starlark.NewBuiltin("url.query_params_encode", core.ErrWrapper(urlModule{}.QueryParamsEncode)),
 				"query_params_decode": starlark.NewBuiltin("url.query_params_decode", core.ErrWrapper(urlModule{}.QueryParamsDecode)),
+
+				"parse": starlark.NewBuiltin("url.parse", core.ErrWrapper(urlModule{}.ParseURL)),
 			},
 		},
 	}
 )
 
 type urlModule struct{}
 
+// urlValue stores a parsed URL
+type urlValue struct {
```

Comment:
_(optional)_ in this project, we're aiming to being careful in our API design _and_ defaulting to exporting primary types.  Given that the `urlValue` is effectively part of the `ytt` Starlark programming context, it's generally useful to export the same type in the Go code.

Suggestion: export `urlValue`

Reply:
I see. So apart of `UrlValue`, should we also export `urlUser` which is also a starlark type?